hello-world-single-file/README.md

```diff
@@ -0,0 +1,15 @@
+# Hello World (single-file)
```

Comment:
I would call this whole sample scratchpad - I think that can be a really useful way to present it to users. "Hey, you want a simple way to develop everything all together while you're experimenting? Here you go."

Reply:
Nice, OK I've renamed it as "scratchpad".